packages/design-system-docs/src/pages/components/Table/_Table.docs.scss

```diff
@@ -20,6 +20,79 @@ Markup: table-complex.example.html
 Style guide: components.table.complex
 */
 
+/*
+`<Table>`
+
+`This is a beta version of the Table components. The functionality and usage is subject to change in response to the feedback from design system users.`
+A `Table` component renders data table using semantic markup and ARIA. The table components allows for customization as needed by the users.
```

Comment:
I can help you on this, but we probably want to give more guidance on how our react table works, as well as the different responsive considerations and options.